tests/foreman/cli/test_ping.py

```diff
@@ -52,7 +52,7 @@ def test_positive_ping(target_sat, switch_user):
     # iterate over the lines grouping every 3 lines
     # example [1, 2, 3, 4, 5, 6] will return [(1, 2, 3), (4, 5, 6)]
     # only the status line is relevant for this test
-    for _, status, _ in zip(*[iter(result.stdout)] * 3):
+    for _, status, _ in zip(*[iter(result.stdout)] * 3, strict=False):  # should this be strict?
```

Comment:
I did some solid archeology and I found that aedae75647180f150ce26801cdeb869e5bdb7270 introduced the `zip` logic. @rmynar is changing it in #13489 anyway.

To answer the question in the comment, I would say that it should not be strict since
```python
(Pdb++) foo = zip(*[iter(result.stdout)] * 3, strict=True)
(Pdb++) for ff in foo: print(ff)
('database:         ', '    Status:          ok', '    Server Response: Duration: 0ms')
('candlepin:        ', '    Status:          ok', '    Server Response: Duration: 17ms')
('candlepin_auth:   ', '    Status:          ok', '    Server Response: Duration: 14ms')
('candlepin_events: ', '    Status:          ok', '    Server Response: Duration: 0ms')
('katello_events:   ', '    Status:          ok', '    Server Response: Duration: 0ms')
('pulp3:            ', '    Status:          ok', '    Server Response: Duration: 77ms')
('pulp3_content:    ', '    Status:          ok', '    Server Response: Duration: 39ms')
('foreman_tasks:    ', '    Status:          ok', '    Server Response: Duration: 3ms')
*** ValueError: zip() argument 2 is shorter than argument 1
```

Reply:
Hmmm... i wonder that's being cutoff by the zip then.
Will change strict to False for this, thanks for digging into this @ogajduse!